mlflow/java/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.mlflow</groupId>
+  <artifactId>mlflow-java</artifactId>
```

Comment:
Kind of a nit but shouldn't we just call it `mlflow`? I guess we can rename it later too.

In the long run we'll probably want this to be a separate artifact from the Java API client anyway, so we'll need to make a more complicated pom.xml.

Reply:
@mateiz I can definitely just name it `mlflow` if you'd prefer. Let me know!

Reply:
@mateiz Maven's package naming conventions, which are derived from Java conventions, indicate that package names should correspond to valid domains that the package owner controls. Since we own `mlflow.org`, `org.mlflow` seems to be the correct package name. 

Here's the Java reference I found regarding package naming: https://docs.oracle.com/javase/specs/jls/se10/jls10.pdf (Page 137)

Reply:
Ah, I wasn't asking about the `org.mlflow` line, but about the `mlflow-java` artifact ID. Why not just `mlflow`.

Reply:
Ah, sorry! I'll correct this in the followup PR that adds SageMaker support to the Java library!